ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -76,13 +76,14 @@ function pushNodes(key, cluster, dataSource, t) {
   ) {
     const nodes = cluster[key].nodes
     dataSource.push({
+      key: key + '(' + nodes.length + ')',
```

Comment:
why key does include `nodes.length`?

Reply:
You're write, It doesn't need. I'll change it.